worker/uniter/modes.go

```diff
@@ -256,20 +284,24 @@ func modeAbideDyingLoop(u *Uniter) (next Mode, err error) {
 	}
 	for {
 		if len(u.relations.GetInfo()) == 0 {
-			return continueAfter(u, newSimpleRunHookOp(hooks.Stop))
+			return continueAfter(u, "running stop hook", newSimpleRunHookOp(hooks.Stop))
 		}
 		var creator creator
+		var userMessage string
 		select {
 		case <-u.tomb.Dying():
 			return nil, tomb.ErrDying
 		case actionId := <-u.f.ActionEvents():
 			creator = newActionOp(actionId)
+			userMessage = fmt.Sprintf("running action %q", actionId)
 		case <-u.f.ConfigEvents():
 			creator = newSimpleRunHookOp(hooks.ConfigChanged)
+			userMessage = "running config-changed hook"
```

Comment:
Is this useful as a pure "user message"? If we're going to expose what we're running, don't we want to make it machine-consumable?